### DISCUSSION TITLE
Why Only Object Storage , no support block device distributed system or storage disk on node ?

### DISCUSSION THREAD

**Discussion by Icedroid** [juicedata/juicefs#4277] [General]
**What would you like to be added**:
A Ceph RGW need at least 3 nodes, If only two nodes have HDD or Nvme disks, how to use juicefs?

**Why is this needed**:
If juicefs support Distributed Storage System block device , it will be great!

**Comment by evgeni-roif-gong** [comment]
I also looking at the options. The [standalone](https://juicefs.com/docs/community/quick_start_guide) guide specify there is a way to use local disk when using the local installation. Meaning I potentially can use AWS EBS storage to host the files. I did not see those options in the helm chart. @Icedroid have you seen anything of that sort to be supported in kubernetes?

**Comment by Cloud-Yao** [comment]
> I also looking at the options. The [standalone](https://juicefs.com/docs/community/quick_start_guide) guide specify there is a way to use local disk when using the local installation. Meaning I potentially can use AWS EBS storage to host the files. I did not see those options in the helm chart. @Icedroid have you seen anything of that sort to be supported in kubernetes?

+1

**Comment by SandyXSD** [comment]
To use distributed block storage as the backend, it requires:

1. The block device can be attached to multiple hosts, and can be read/written by them at the same time. However, most of the distributed block device can be written by only ONE node.
2. There is a distributed management system for the block device, to handle cache/locks among multiple hosts to achieve consistency. This means local file systems (ext4, xfs, etc.) are not applicable.
